### DISCUSSION TITLE
Getting all records from hasMany records [Apr 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1922] [Discord Threads]
> 🙋‍♂️ **dave_see**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### What version are you on ? If < 0.90 - please upgrade to the latest ones.   
####    
#### The APIs are slightly changed but you will get all records from hasMany relation  
#  
> 💁 **o1lab**  
#### apis.nocodb.com has the details  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/966407803017240606)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
